extensions/amp-story-audio-sticker/validator-amp-story-audio-sticker.protoascii

```diff
@@ -0,0 +1,247 @@
+tags: {  # amp-story-audio-sticker
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-story-audio-sticker"
+    version: "0.1"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-story-audio-sticker>
+  html_format: AMP
+  tag_name: "AMP-STORY-AUDIO-STICKER"
+  requires_extension: "amp-story-audio-sticker"
+  mandatory_parent: "AMP-STORY-GRID-LAYER"
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: CONTAINER
+  }
+  attrs: {
+    name: "size"
+    value: "small"
+    value: "large"
+  }
+  attrs: {
+    name: "sticker"
+    value: "cat-sticker"
```

Comment:
This value looks like it duplicates the "sticker" naming twice, in the attr and value. Should this be just "cat"? The other values don't say sticker at the end